commit.go

```diff
@@ -167,17 +167,17 @@ var (
 // variable, if it's set.  The contents are expected to be a JSON-encoded
 // github.com/openshift/api/config/v1.Image, set by an OpenShift build
 // controller that arranged for us to be run in a container.
-func checkRegistrySourcesAllows(forWhat string, dest types.ImageReference) error {
+func checkRegistrySourcesAllows(forWhat string, dest types.ImageReference) (bool, error) {
```

Comment:
Just for clarity, it might be good to name the return values if gofmt allows:
```suggestion
func checkRegistrySourcesAllows(forWhat string, dest types.ImageReference) (insecure bool, err error) {
```